portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase

```diff
@@ -94,34 +94,19 @@ definition {
 			Category.openCategoriesAdmin(siteURLKey = "global");
 		}
 
-		task ("Assert if Vocabulary is for internal use only") {
-			AssertElementPresent(
-				key_vocabularyName = "Audience",
-				locator1 = "Categories#VOCABULARY_ICON_INTERNAL_USE");
-
-			AssertElementPresent(
-				key_vocabularyName = "Stage",
-				locator1 = "Categories#VOCABULARY_ICON_INTERNAL_USE");
-		}
-
-		task ("Assert if Vocabulary allow multiple categories") {
-			Vocabulary.viewVocabularyAllowMultipleCAtegories(
-				label = "Allow Multiple Categories",
-				vocabularyName = "Audience");
-
-			Vocabulary.viewVocabularyAllowMultipleCAtegories(
-				label = "Allow Multiple Categories",
-				vocabularyName = "Stage");
-		}
-
-		task ("Assert if Vocabulary allow multiple categories") {
-			Vocabulary.viewVocabularyAssociatedAssetTypesRequirement(
-				label = "Required",
-				vocabularyName = "Audience");
-
-			Vocabulary.viewVocabularyAssociatedAssetTypesRequirement(
-				label = "Required",
-				vocabularyName = "Stage");
+		task ("Assert configurations for vocabularies") {
+			for (var vocabularyName : list "Audience,Stage") {
+				AssertElementPresent(
```

Comment:
Try not to use "for" loops in the testcase file. If you are using a for loop, maybe you need a macro for that :)

Reply:
@Luizcarlosqueiroz review this please

Reply:
Alright. I was not using a loop before, and then I thought that it would be a shorter code. But I'll change back as how it was.